addons.conf

```diff
@@ -24,6 +24,11 @@ server {
         alias /srv/user-media/guarded-addons/;
     }
 
+    location /code/storage/sitemaps/ {
+        internal;
+        alias /srv/user-media/sitemaps/;
```

Comment:
> `SITEMAP_STORAGE_PATH` in addons-server does not seem to point to this path. 

None of the paths point to `/user-media/` (e.g. `guarded-addons` above is similarly stored in `<storage_root>/guarded-addons` locally).  It just needs to match the path in `docker-compose.yml` - I'm unfamiliar with how it will need to be defined in ops managed nginx though.

> Also, is a sitemap "user media"?

it's not strictly `user-media` - it was easier to not have yet another /path/ on the root on prod and I figured it wouldn't hurt.  But I could change it?

Reply:
Maybe add a comment pointing out that it matches the volume mount set in `docker-compose.yml` in addons-server to clarify.

Reply:
https://github.com/mozilla/addons-nginx/pull/19/commits/bbd7f485914353c8e47251161e1e54b37241c3ec